apache/superset
# chore(dashboard): Resolve form_data vs. formData in dashboard redux state

### SUMMARY

This PR cleans up the Redux state for the Dashboard view by combining `charts[i].formData` and `charts[i].form_data` into just `charts[i].form_data`.  It also does some minor typing adjustments to support the change.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS

No functionality should be impacted by this PR, so try out as much Dashboard functionality as possible, especially editing dashboards, and make sure nothing behaves differently than before.

### ADDITIONAL INFORMATION
